update airnode packages

Ashar2shahid opened the pull request:
update airnode packages to v0.7.2
also updated configs for new airkeeper release

acenolaza commented on src/create-config.ts:
```diff
@@ -76,7 +76,7 @@ const buildNodeSettings = (
     logFormat: 'plain' as const,
     logLevel: 'INFO' as const,
-    stage: `prod-${timestamp}`,
+    stage: `prod${timestamp}`,
```
Is this intentional? Do you mind sharing the requirement for this change?

Ashar2shahid replied:
there was issue which burak bought up about gcp having issues when stage has a '-' in it

bbenligiray replied:
This is not a thing, please revert. I said "changing `stage`, for example, by adding a `-` at the end fixes the problem". It's not about the symbol specifically.

Ashar2shahid replied:
got it

acenolaza commented on src/create-config.ts:
```diff
@@ -358,7 +360,7 @@ const main = async (operationRepositoryTarget?: string) => {
       [template.templateId]: {
         endpointId: template.endpointId,
-        templateParameters: template.parameters,
+        encodedParameters: template.parameters,
```
I'm guessing we will need to merge the Airkeeper dependencies update [PR](https://github.com/api3dao/airkeeper/pull/64) to main. I was hoping to get a second review from Michal when he's back but Vekil has already approved it.

acenolaza reviewed (commented):
The operations/src/airkeeper-types.ts types are out of date and I have a quick question. Does it make sense to update old config files for existing deployments? Shouldn't we just leave them as is and create new folders with a new timestamp?

Ashar2shahid commented:
btw, I didn't change anything in the old deployment files. validation was giving a prettier error. So its just a formatting change

acenolaza commented:
> btw, I didn't change anything in the old deployment files. validation was giving a prettier error. So its just a formatting change

I meant that the types for Airkeeper will cause the build step to fail due to references in airnode-node package that have changed.

Ashar2shahid commented:
Reverted the "-" changes and fixed the airkeeper types with the latest version. Waiting for the official 0.1.0 airkeeper deployment on npm before importing it into operations
